### PR TITLE
Fix misspellings

### PR DESCRIPTION
**Pull request by creydr**
**What this PR does / why we need it**:

This PR fixes some misspellings in comments and messages. No big issues, but I found some in error messages so fixed the other ones as well (got the fill list from [goreportcard.com](https://goreportcard.com/report/github.com/kubevirt/kubevirt#misspell)).

Afterwards I run a `make generate` to update the generated files as well (a `make test` worked too).

To prevent this in future it could make sense to have a spell check like the `misspell` linter in [`golangci-lint`](https://golangci-lint.run/) (e.g. as a Github action).

**Release note**:
```release-note
"NONE"
```

### CODE REVIEW

**Comment by rmohr** [comment]
/lgtm
/approve

Thanks a lot! :+1:

**Comment by rmohr** [comment]
@creydr thanks for the hint with `misspell`. I see that for instance kubernetes/test-infra uses it via bazel: https://github.com/kubernetes/test-infra/search?q=misspell. We could just add it to our regular linter checks like this, which we already run.

**Comment by creydr** [comment]
@rmohr could you please check again? I had to resolve a merge confict meanwhile.

**Comment by creydr** [comment]
Can somebody please tigger a /retest ?

**Comment by creydr** [comment]
Can somebody please tigger a /retest again?
Thanks

**Comment by creydr** [comment]
rebased it

**Comment by creydr** [comment]
rebased it

File: tests/vmi_configuration_test.go [hunk 1]
```diff
@@ -1292,8 +1292,13 @@ var _ = Describe("Configurations", func() {
 				By("Expecting the VirtualMachineInstance console")
 				Expect(tests.LoginToFedora(agentVMI)).To(Succeed())
 
+<<<<<<< HEAD
 				By("Terminating guest agent and waiting for it to dissappear.")
 				Expect(console.SafeExpectBatch(agentVMI, []expect.Batcher{
+=======
```
Comment (rmohr) [inline tests/vmi_configuration_test.go:1298]: Oh a merge leftover.

**Review by rmohr — CHANGES_REQUESTED** [review]
/lgtm cancel

File: tests/vmi_configuration_test.go (hunk 1 above)
Comment (creydr) [reply tests/vmi_configuration_test.go]: Oh no. Sorry I didn't notice this. Fixed it now.

**Comment by creydr** [comment]
rebased it

**Comment by creydr** [comment]
Can somebody please trigger a /retest?
Thanks

**Comment by rmohr** [comment]
Now that was a bumpy merge experience. @creydr thanks for staying with us. :+1:

**Comment by creydr** [comment]
@rmohr: you are very welcome. Thanks for your support

**Pull request merged**